### DISCUSSION TITLE
这周你做了什么？2022-06-24

### DISCUSSION THREAD

**Discussion by jiacai2050** [EmacsTalk/emacstalk.github.io#22] [Show and tell]
- 可以随便讨论本周你打算做什么与 Emacs 相关的事情，甚至可以寻求帮助或反馈。
- 请记住，什么都不做也没关系！
- Happy Hacking Emacs！

**Comment by jiacai2050** [comment]
- 准备周末剪辑、发布第十三期
  - https://github.com/EmacsTalk/joinus/blob/master/podcasts/2022-04-08-yaoni.org
- Emacs china 目前访问有些问题，子龙说预计周日能恢复，大家不要捉急。

**Comment by beyondpie** [comment]
攒了一套尽量基于原生emacs的小配置，用以支持远程服务器的环境。
目前在学习emacs自己的version control，计划不使用magit，而复杂的操作就放在terminal里面直接运行。
